projects/dsp-ui/src/lib/search/search-panel/search-panel.component.html

```diff
@@ -44,9 +44,8 @@ <h4 *ngIf="!showAdvanced">Expert search</h4>
             </span>
         </div>
         <div class="dsp-menu-content">
-            <!-- TODO: reactivate this tags when the components are ready -->
-            <!-- <dsp-advanced-search *ngIf="showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-advanced-search> -->
-            <!-- <dsp-expert-search *ngIf="!showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-expert-search> -->
+            <dsp-advanced-search *ngIf="showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-advanced-search>
+            <dsp-expert-search *ngIf="!showAdvanced" (search)="search.emit($event); closeMenu()"></dsp-expert-search>
```

Comment:
Shouldn't this check for the input `expert`?

Reply:
I do not understand what you mean but the flag showAdvanced is doing the check (advanced or expert search).

Reply:
But couldn't you use [`*ngIf` with an else block](https://angular.io/api/common/NgIf#description) to make this clearer?

Reply:
I was just wondering whether the `*ngIf ... else` syntax would be clearer than two `*ngIf`s.

Reply:
I would keep it this way